frontend/packages/dev-console/src/components/add/__tests__/SampleGettingStartedCard.spec.tsx

```diff
@@ -38,7 +38,22 @@ jest.mock(
 const useActiveNamespaceMock = useActiveNamespace as jest.Mock;
 const CatalogServiceProviderMock = CatalogServiceProvider as jest.Mock;
 
+afterEach(() => {
+  delete window.SERVER_FLAGS.addPage;
+});
+
 describe('SampleGettingStartedCard', () => {
+  it('should not be disabled when Samples add card is disabled', () => {
```

Comment:
it('should not render when Samples add card is disabled', () => {

Reply:
Changed, thanks.